.github/workflows/master.yaml

```diff
@@ -37,11 +37,11 @@
             },
             {
                "name": "Protobuf generation",
-               "run": "find . bazel-bin/pkg/proto -name '*.pb.go' -delete || true\nbazel build $(bazel query --output=label 'kind(\"go_proto_library\", //...)')\nfind bazel-bin/pkg/proto -name '*.pb.go' | while read f; do\n  cat $f > $(echo $f | sed -e 's|.*/pkg/proto/|pkg/proto/|')\ndone\n"
+               "run": "if [ -d bazel-bin/pkg/proto ]; then\n  find . bazel-bin/pkg/proto -name '*.pb.go' -delete || true\n  bazel build $(bazel query --output=label 'kind(\"go_proto_library\", //...)')\n  find bazel-bin/pkg/proto -name '*.pb.go' | while read f; do\n    cat $f > $(echo $f | sed -e 's|.*/pkg/proto/|pkg/proto/|')\n  done\nfi\n"
```

Comment:
(not for this change, but) this is getting a bit hard to read, and I'd love to send a follow-on to fix that. What direction do you all prefer?
* moving contents into a dedicated script; workflow calls the script
* moving workflows to be generated from Jsonnet (since I guess they aren't already); the script can be inlined in a readable way

I'd prefer the first, so that humans can use the same scripts to re-generate proto files easily, but I think other buildbarn repos take the latter approach